### DISCUSSION TITLE
Slow loading theme

### DISCUSSION THREAD

**Discussion by jmgb27** [bootscore/bootscore#503] [Q&A] [answered]
Hi, Is it just me or the theme became slow recently?, I have tried other themes and they are fast, but when I switch to bootscore, it slows down the page load again. I have tried debugging, I have tried to comment out wp_head() and it loads faster probably because there are no scripts when you commented that out. 

I have tried also downloading bootscore fresh new version and also older versions. but still they are loading slow on whole pages (local hosted, and the page loading have an average of 6-7 seconds!!), except for the wp-admin dashboard, it loads fast.

**Comment by crftwrk** [comment, marked as answer]
Hi,

do you have enabled the developer mode? If so, SCSS gets recompiled on each page load and this will take 6-7 seconds.

Open wp-config.php and checks if following line is there:

```php
define('WP_ENVIRONMENT_TYPE', 'development'); 
```

Delete it.

See https://bootscore.me/documentation/theme/bootstrap-css-sass/#Developer_mode

**Reply by jmgb27** [reply]
It works!, thank you very much

**Reply by crftwrk** [reply]
You‘re welcome
